packages/application-config/src/process-config.ts

```diff
@@ -129,6 +129,7 @@ const processConfig = ({
                 ? undefined
                 : appConfig.env.development.initialProjectKey,
             teamId: appConfig.env.development?.teamId,
+            applicationId: appConfig.env.development?.applicationId,
```

Comment:
Should we validate that the `applicationId` is provided if the `teamId` is used?